Bayesian_Cognitive_Modeling/ParameterEstimation/DataAnalysis/Planes.stan

```diff
@@ -0,0 +1,42 @@
+// #### Notes to Stan model #######################################################
+// ## Stan has hypergeometric distribution implemented, so in one way the code
+// ## is more intuitive. On the other hand, Stan can't sample discrete parameters,
+// ## therefore we have to increment log probability manually (as we did in 
+// ## Survey example).  
+// ################################################################################
+
+// Planes
```

Comment:
Drop this comment---it's outside the data block and it's not at all clear what it means!